runtime/oti/j9nonbuilder.h

```diff
@@ -5844,8 +5914,8 @@ typedef struct J9JavaVM {
 #endif /* (JAVA_SPEC_VERSION >= 16) || defined(J9VM_OPT_CRIU_SUPPORT) */
 #define J9JAVAVM_REFERENCE_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(U_32) : sizeof(UDATA))
 #define J9JAVAVM_OBJECT_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9ObjectCompressed) : sizeof(J9ObjectFull))
-#define J9JAVAVM_CONTIGUOUS_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9IndexableObjectContiguousCompressed) : sizeof(J9IndexableObjectContiguousFull))
-#define J9JAVAVM_DISCONTIGUOUS_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9IndexableObjectDiscontiguousCompressed) : sizeof(J9IndexableObjectDiscontiguousFull))
+#define J9JAVAVM_CONTIGUOUS_INDEXABLE_HEADER_SIZE(vm) (vm->contiguousIndexableHeaderSize)
+#define J9JAVAVM_DISCONTIGUOUS_INDEXABLE_HEADER_SIZE(vm) (vm->discontiguousIndexableHeaderSize)
```

Comment:
put extra pair of oval brackets (same for vmThread based macros):
(vm)->discontiguousIndexableHeaderSize